doc/themes/scikit-learn/static/nature.css_t

```diff
@@ -1389,3 +1389,12 @@ li#other-versions {
   }
 
 }
+
+/* --- Glossary ------------------------------------------------------------- */
+
+code.std-term {
+    font-family: inherit;
+    font-size: 1em;
+    font-weight: inherit;
+    padding: 0;
+}
```

Comment:
Was there the idea to do this from the python side instead of with css? 
(in any case, otherwise some more comments here to explain it would be welcome)

Reply:
This PR was updated with a python solution. It forces `any` roles that resolve to `term` to have the same html structure as a normal `term` role.